odoo/modules/neutralize.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import textwrap
+
+import odoo
+from .module import get_manifest
+
+
+def get_installed_modules(cursor):
+    cursor.execute('''
+        SELECT name FROM ir_module_module
+        WHERE state IN ('installed', 'to upgrade', 'to remove', 'to install');
```

Comment:
Modules in state `to_install` are not installed yet
```suggestion
        WHERE state IN ('installed', 'to upgrade', 'to remove');
```